src/pack.iced

```diff
@@ -1,12 +1,14 @@
-try
-  mp = require 'msgpack'
-catch e
 
-if not mp?
-  try
-    pp = require 'purepack'
-    mp = pp
-  catch e
+# We want purepack only
```

Comment:
we could theoretically do the require 'msgpack' + '' to work around the require issue but is there even a reason to use non-purepack?